usrsctplib/CMakeLists.txt

```diff
@@ -188,6 +188,10 @@ if(SCTP_USE_MBEDTLS_SHA1)
 	target_include_directories(usrsctp PRIVATE ${MBEDTLS_INCLUDE_DIRS})
 endif()
 
+if(APPLE)
+	target_compile_definitions(usrsctp PRIVATE __APPLE_USE_RFC_2292)
+endif()
+
```

Comment:
Why doesn't work https://github.com/sctplab/usrsctp/blob/master/usrsctplib/CMakeLists.txt#L87 for you?

Reply:
Ah, I didn't notice that one. You asked me to move the fix to the build system so I did. Your change should work just as fine I expect.